### ISSUE TITLE
Pathinfo data gone since upgrading to 1.0

### ISSUE THREAD

**Issue by jasonvarga** [thephpleague/flysystem#391]
Hi, I've just upgraded from 0.5 to 1.0 and have noticed some differences.

In 0.5, I used `listFiles($path)` which returned something like this:

``` php
'dirname'   => 'path/to/folder',
'basename'  => 'myfile.txt',
'extension' => 'txt',
'filename'  => 'myfile',
'path'      => 'path/to/folder/myfile.txt',
'type'      => 'file',
'timestamp' => 1421811561,
'size'      => 123
```

Now in 1.0 I see this:

``` php
'type'      => 'file',
'path'      => 'path/to/folder/myfile.txt',
'timestamp' => 1421811561,
'size'      => 123
```

I'm using the Local adapter.  It seems like the data you'd find by using `pathinfo` is now gone.  
In my case, I was relying on the `filename` key for something, and now it's gone.

How come we now see less data returned? Did I do something incorrectly when upgrading or do I need to parse the filename using `pathinfo` manually now?

Thanks!

**Comment by frankdejonge** [comment]
@jasonvarga I've released 1.0.1 to include this fix.

**Comment by jasonvarga** [comment]
Awesome, thank you.
